code/backend/batch/utilities/helpers/EnvHelper.py

```diff
@@ -83,7 +83,8 @@ def __init__(self, **kwargs) -> None:
         # Otherwise, keys are obtained from environment variables.
         if self.AZURE_AUTH_TYPE == "rbac":
             self.AZURE_SEARCH_KEY = None
-            self.AZURE_OPENAI_KEY = ""
+            self.AZURE_OPENAI_KEY = ""  # TODO: See if this is still used.
```

Comment:
Left over TODO

Reply:
I was scared to remove it! Will do some testing